package.json

```diff
@@ -273,6 +284,10 @@
           "command": "vscode-docker.browseDockerHub",
           "when": "view == dockerExplorer && viewItem == dockerHubNamespace"
         },
+        {
+          "command": "vscode-docker.queueBuild",
+          "when": "view == dockerExplorer && viewItem == localImageNode"
```

Comment:
If user can queue build from image node, how can they select the source location and docker file? If we want to enable this entry, we should allow uses to select the source location and docker file.